testgrid/config/config.yaml

```diff
@@ -1039,10 +1039,12 @@ dashboards:
   - name: gce-ha-master
     test_group_name: kubernetes-e2e-gce-ha-master
 
-- name: google-gci
+# "google-gci-dev" is a private tab that should be looked at by GCI/COS
+# developers. Tests under this tab are used for GCI/COS image development.
```

Comment:
I still like to include "development" because the gci-ci-* jobs are to qualify a newer k8s version before it's builtin to GCI, as opposed to qualifying a GCI image before release. So I made it "development AND qualification".